### ISSUE TITLE
Record dates become scrambled after using Watsonian Vice-county filter

### ISSUE THREAD

**Issue by sophiathirza** [nbnuk/nbnatlas-issues#378]
This is a second issue from forum post: (https://forums.nbn.org.uk/viewtopic.php?pid=27546#p27546).

I then tried putting the Watsonian Vice-county filter for the Outer Hebrides on, and the record dates became scrambled starting at 01-01-1670 and ending with 01-01-9999.

**Comment by sophiathirza** [comment]
I can't recreate this. @EllaVogel and @CEJohnston could you try and recreate it, please?

**Comment by CEJohnston** [comment]
I have managed to replicate.  
I created a random polygon round the Outer Hebrides (in Search by Polygon) then switched on the Vice County layer as a filter and filtered by the VC. I got 913441 records and have dates from 1629 and later in the 17th century, as well as 9999.  The 01-01-1629 records are in the BLS Mapping Scheme dataset; the 01-01-9999 records are in the Standing Waters Database - Scotland dataset (published by SNH).  
Just looking at the datasets the Standing Waters - Scotland dataset has 519 records with a 9999 year (these are dubious!); the BLS Mapping Scheme dataset says it dates from 1750-2009 but all its records are either dated  01-01-1629 (16,806) or 01-01-1960 (477,442) (looks dubious too).
The Outer Hebrides 01-01-1670 records (182 records in my search) come from two datasets: Mollusc (non-marine): 1999 Atlas Dataset for Great Britain and Ireland, and Mollusc (non-marine): Compilation of records of rare and scarce species for Great Britain and Ireland.  They may be genuine but they may not.

**Comment by sophiathirza** [comment]
Thanks for investigating.

Standing Waters - Scotland: 01-01-9999 (I think) indicates where the date was not supplied, 9999 was given as the date. These should not be shown in the phenology charts. Sometimes the occurrence record says 'date not supplied' and other times just gives the date and gives the date precision as Day.

BLS Mapping Scheme: the metadata says that they only distinguish records from before and since 1960. So having just the two dates in the dataset make sense, although not very clear.

Mollusc (non-marine): 1999 Atlas Dataset for Great Britain and Ireland: it looks like some of these are digitised records, where the date range is 1629-31/12/1950. They have probably been included because their Atlas has data from 1876-1999.

In summary, the dates have not got scrambled on filtering, it is how they were uploaded to the Atlas.

**Comment by sophiathirza** [comment]
Raised issue to have date not supplied precision: #384
